src/main/java/seedu/address/logic/commands/AddMemberCommand.java

```diff
@@ -14,34 +11,28 @@
 /**
  * Adds a person to the address book.
  */
-public class AddCommand extends Command {
+public class AddMemberCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
```

Comment:
```suggestion
    public static final String COMMAND_WORD = "addm";
```
I think the command should be `addm`.